docs/plugins/cloud-gce.asciidoc

```diff
@@ -46,6 +46,48 @@ discovery:
       type: gce
 --------------------------------------------------
 
+
+[IMPORTANT]
+.Binding the network host
+==============================================
+
+It's important to define `network.host` as by default it's bound to `localhost`.
+
+You can use {ref}/modules-network.html[core network host settings] or
+<<discovery-gce-network-host,gce specific host settings>>:
+
+==============================================
+
+[[discovery-gce-network-host]]
+==== GCE Network Host
+
+When the `cloud-gce` plugin is installed, the following are also allowed
+as valid network host settings:
+
+[cols="<,<",options="header",]
+|==================================================================
+|GCE Host Value |Description
+|`_gce:privateIp_` |The private IP address of the machine.
+|`_gce:hostname_` |The hostname of the machine.
+|`_gce_` |Less verbose option for the private ip address.
+|==================================================================
+
+By default, GCE discovery will use network card `0` when fetching `_gce:privateIp_`
```

Comment:
can we try and use google terminology here? they call this network interface. Also - are we sure `0` is always/95% of the time is there? I'm afraid that the OOB behavior will be bad..